snapshot-labs/snapshot
# feat: resolve handle using stamp

### Summary

From https://github.com/snapshot-labs/pitches/issues/58

And following https://github.com/snapshot-labs/snapshot/pull/4375

Use stamp API to resolve handles (converting domain name to address)

### How to test

1. Go to https://snapshot-git-resolve-handle-using-stamp-snapshot.vercel.app/#/polls.lenster.xyz/proposal/0xda179d73d85797602a8f788eb8a326cda541a9a23d6b6b6e785682f7a98ec93e#/
2. in the votes section, "Load more votes"
3. in the modal's search bar, input some domains (lens, eth, and other)
4. it should resolve the handles using stamp API

### To-Do

- [x] Waiting for https://github.com/snapshot-labs/snapshot/pull/4375 to be merge
- [x] Waiting for https://github.com/snapshot-labs/stamp/pull/115 to be merged
- [ ] OPTIONAL - debounce the search bar (it currently send an API request on each character)
